src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -54,6 +97,15 @@ public void toggleIntake(boolean reverse) {
     }
   }
 
+  public void deployIntake() {
```

Comment:
This is set up as a toggle.  

Not sure we want this.  Feel like we should have deployIntake and retractIntake, with a third method that returns a boolean when the intake is deployed.  

This way, auton can have deployIntake, and be certain that it is not going to toggle anything, but will absolutely deploy.

If we change the button from a whenpressed to a whileheld, we can specify exactly what happens.  retract upon initialize, nothing during execute, and deploy at end.

And if you end up needing a toggle, the code is still easy:

if ( intakeDeployed() )
  retractIntake()
else
  deployIntake()